mailer.rst

```diff
@@ -214,6 +214,15 @@ OhMySMTP             ohmysmtp+smtp://API_TOKEN@default                    n/a
         The usage of ``default_socket_timeout`` as the default timeout was
         introduced in Symfony 5.1.
 
+.. note::
+
+    To use Google Gmail, you must have a Google Account with 2-Step-Verification (2FA)
+    enabled and you must use `App Password`_ to authenticate. Also note that Google
+    revokes your App Passwords when you change your Google Account password and then
+    you need to generate a new one.
+    Using other methods (like ``XOAUTH2`` or the ``Gmail API``) are not supported currently.
+    You should use Gmail for testing purposes only and use a real provider in production. 
```

Comment:
I would remove this last sentence as we already have such a warning in this doc.

Reply:
Done in 81f513755